nemo_curator/datasets/doc_dataset.py

```diff
@@ -43,6 +43,8 @@ def read_json(
         files_per_partition: int = 1,
         add_filename: bool = False,
         input_meta: Union[str, dict] = None,
+        columns: List[str] = None,
```

Comment:
nit on the type hint, it should be `Optional[List[str]]` given default is `None`

Reply:
Thanks, updated!